requirements.txt

```diff
@@ -24,7 +24,8 @@ money
 pymarc
 accept-types
 watchtower # for cloudwatch logging
-pyspellchecker
+pyspellchecker==0.5.6
+parameterized
```

Comment:
`metadata_wrangler` doesn't use this package but `server_core` does. The tests on Travis failed though because it couldn't find this module. I thought it would recursively install all modules from all `requirements` files, no?

Reply:
You can get that behavior by sticking `-r core/requirements.txt` at the top of this requirements.txt file.

Reply:
Thanks. I'll leave it like this for now because there are a lot of overlapping modules which will most likely get updated with the python 3 update.